### PR TITLE
Inline examples: Use "longitude" and "latitude" instead of "x" and "y" and add units in comments

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR aims to make comments of the inline examples more precise, by using `longitude` and `latitude` in stead of `x` and `y` for geographic grids and adding units.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/src/grdclip.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ def grdclip(grid, **kwargs):
     Example
     -------
     >>> import pygmt
-    >>> # Load a grid of @earth_relief_30m data, with an x-range of 10 to 30,
-    >>> # and a y-range of 15 to 25
+    >>> # Load a grid of @earth_relief_30m data, with a longitude range of
```
Comment (seisman) [inline pygmt/src/grdclip.py:86]: Actually, I'm not sure if we should have the `°` symbol in the examples. It's a non-ASCII symbol.

Comment (seisman) [reply pygmt/src/grdclip.py:86]: This is how the `grdhisteq` docstrings look like if opened with "GB 2312" encoding. It looks pretty bad.
Screenshot from 2023-10-12 16-10-14

GB 2312 is the default encoding for the Windows Chinese version, but I'm not sure about the default encoding when VSCode opens a plaintext file.

Comment (yvonnefroehlich) [reply pygmt/src/grdclip.py:86]: Yeah, that looks not good, and we should change this to either not using the `°` symbol or writing `deg`.

Comment (seisman) [reply pygmt/src/grdclip.py:86]: I prefer to "10 E", instead of "10 deg E".

Comment (yvonnefroehlich) [reply pygmt/src/grdclip.py:86]: I have updated this in commit https://github.com/GenericMappingTools/pygmt/pull/2740/commits/88d4b751ea011945b44c966f71e0e4ba45351605.

Comment (seisman) [reply pygmt/src/grdclip.py:86]: Sorry, after thinking twice, I think I've changed my mind. These non-ASCII characters are rendered correctly on the PyGMT site, so they won't affect most users.

Users who open PyGMT source codes in their text editor should know that UTF-8 is always the best and default encoding for Python codes (actually for any codes).

I feel we should revert the changes in 88d4b751ea011945b44c966f71e0e4ba45351605.

Comment (yvonnefroehlich) [reply pygmt/src/grdclip.py:86]: No worries, that's fine with me. I reverted commit https://github.com/GenericMappingTools/pygmt/commit/88d4b751ea011945b44c966f71e0e4ba45351605 in commit https://github.com/GenericMappingTools/pygmt/pull/2740/commits/df55aeebff1fff2bb0f028d3ce00f55b7d9cff9b.

**Review by seisman — APPROVED** [review]

**Pull request merged**